examples/all-clusters-app/all-clusters-common/gen/IMClusterCommandHandler.cpp

```diff
@@ -6380,8 +6380,8 @@ void DispatchServerCommand(app::Command * apCommandObj, CommandId aCommandId, En
 void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aCommandId, chip::EndpointId aEndPointId,
                                   chip::TLV::TLVReader & aReader, Command * apCommandObj)
 {
-    ChipLogDetail(Zcl, "Received Cluster Command: Cluster=%" PRIx16 " Command=%" PRIx8 " Endpoint=%" PRIx8, aClusterId, aCommandId,
-                  aEndPointId);
+    ChipLogDetail(Zcl, "Received Cluster Command: Cluster=%" PRIx32 " Command=%" PRIx32 " Endpoint=%" PRIx16, aClusterId,
```

Comment:
cluster ID's are 16-bit - commands and endpoints 8-bit 
- see https://github.com/project-chip/connectedhomeip/blob/ed677f04ae8187098130f3701b5583f8db56a55e/src/app/util/basic-types.h#L33

Why the change into 32-bit/16-bit prints ?

Reply:
> One thing that I have note that from the spec that I'm not sure of is the fact that `ClusterId` is now a `uint32` , as well as `AttributeId`.
Sorry - now read your comment here. 
I expect 32-bit would create an significant code/frame size impact. 
@tcarmelveilleux Would you know the spec reasoning for these sizes ?

Reply:
> One thing that I have note that from the spec that I'm not sure of is the fact that `ClusterId` is now a `uint32` , as well as `AttributeId`.

I asked around to double check this is not a typo in the spec. The move to `uint32` for `Attribute Id`, `Command Id` and `Cluster Id` is expected. The move of `uint16` for `Endpoint Id` is expected too.

Reply:
I do see the 32-bit is actually a combination of: 16-bit 'Namespace ID' | 16-bit Cluster ID. (10.6.2.5 - v0.7 ballot version)
It's noted that for standard CHIP clusters the Cluster ID can compress down to 16-bit. (namespace being all 0x0000 in that case)
Is it an option to handle these vendor specific clusters separately ? The bulk of the cluster handling isn't using the top part probably.

Reply:
A question for follow-up.
The current ZCL handling code already seems to have a fully separate code path for manufacturer specific cluster handling.
It would avoid using the 32-bit in 90% of the cases where the top 16-bit is all 0x0000 = a standard cluster.

Reply:
That's because in ZCL you can't have manufacturer-specific granularity finer than "entire cluster"....